eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -232,151 +230,4 @@ function Upload-Blobs
     }
 }
 
-if ($Language -eq "javascript")
```

Comment:
I assume all the language repo's have been updated already with any changes made to these deleted lines?

Reply:
Yes, I did another check yesterday to make sure. And there has been no changes since then.